azext_iot/common/utility.py

```diff
@@ -619,3 +619,23 @@ def is_valid_dtmi(dtmi):
     if not pattern.match(dtmi):
         return False
     return True
+
+
+def generate_container_sas_token(storage_cstring):
```

Comment:
Good idea providing this function as a general utility but as its currently written it will only produce fully entitled tokens for 1 hour.

Reply:
Placing the function in the common utility module implies it should be generically useful. There are more needs for storage sas generation than device identity import/export, one example off the top of my head is for ADU where [3 hour or more](https://learn.microsoft.com/en-us/azure/iot-hub-device-update/import-update#if-youre-importing-via-apis-instead) expiry sas tokens are recommended with read permission.

If this function is meant to be narrowly focused on the device identity import/export use case then it shouldn't be a positioned as generic utility.

Furthermore it seems unlikely all those AccountSasPermissions are needed for import export. Ideally only the needed permissions are used to generate a sas token.